### PR TITLE
Why do astronauts use Linux?

### PR DESCRIPTION
**Pull request by developersteve**
added 2 new jokes

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -949,3 +949,15 @@ Lisp programmers don't make prank calls. They make `FUNCALL`s
 **A**: GUI
 
 ---
+
+**Q**: Why do astronauts use Linux?
+
+**A**: They can't open Windows in space!
```
Comment (styfle) [inline readme.md:955]: This one is good!

File: readme.md [hunk 2]
```diff
@@ -949,3 +949,15 @@ Lisp programmers don't make prank calls. They make `FUNCALL`s
...
+
+---
+
+**Q**: Do you know why ARM chips dont get used much for servers? 
+
+**A**: No one wants a servered ARM
```
Comment (styfle) [inline readme.md:961]: This one is...okay

Comment (styfle) [reply readme.md:961]: We already have a couple ARM jokes

Comment (Zo-Bro-23) [reply readme.md:961]: I like this one 😊

**Review by Zo-Bro-23 — APPROVED** [review]

**Pull request merged**